sklearn/impute/_iterative.py

```diff
@@ -29,6 +29,16 @@ class IterativeImputer(_BaseImputer):
     A strategy for imputing missing values by modeling each feature with
     missing values as a function of other features in a round-robin fashion.
 
+    Using defaults, the imputer scales in :math:`\mathcal{O}(knp^3\min(n,p))`
```

Comment:
I would prefer to have this part in the proper `Notes` section below.
In the long description, we can refer to the `Notes` section for information regarding the complexity of the algorithm.